config/keyqueue.js

```diff
@@ -226,6 +300,13 @@ exports.keyqueueReacted = async function(msg, emoji, member) {
                                 color: 0x00ab30
                             }
                         });
+
+                        if (CONFIG.SystemConfig.servers[msg.guildID].keyqueue.keyping) {
+                            let pingChannel = await CONSTANTS.bot.getChannel(CONFIG.SystemConfig.servers[msg.guildID].keyqueue.pingchannel);
+                            let pingRole = CONFIG.SystemConfig.servers[msg.guildID].keyqueue.pingrole;
+
+                            await pingChannel.createMessage(`<@&${pingRole}> ${member.mention} has ${numItems} <:${emojiText}> to pop!`);
```

Comment:
This needs to be changed:

Please verify that pingChannel and pingRole are actually correct. You don't want the bot to try to ping a channel or role that doesn't exist without at least `catching` the error.

There are two ways you can do this:
1. Fetch pingChannel and pingRole asynchronously, and await the result. if either one of them isn't valid, send a message saying so.
2. Or, wrap this block in a try-catch of its own.

Reply:
Fixed in 6c116e65c10340340660dcc8b2311c076acaec87